src/shared/components/query/QueryStore.ts

```diff
@@ -1162,6 +1170,9 @@ export class QueryStore
 				return "Expression specified in the list of genes, but not selected in the Molecular Profile Checkboxes.";
 
 		}
+		if(this.isVirtualCohortQuery && !(this.dataTypePriority.mutation || this.dataTypePriority.cna)){

```

Comment:
this also should have a test.  these are the new rules : )

Reply:
done!